k9mail/src/main/java/com/fsck/k9/helper/HtmlConverter.java

```diff
@@ -1323,6 +1323,30 @@ private static String htmlifyMessageFooter() {
         return "</pre>";
     }
 
+    public static String wrapStatusMessage(CharSequence status) {
+        return wrapMessageContent("<div style=\"text-align:center; color: grey;\">" + status + "</div>");
+    }
+
+    public static String wrapMessageContent(CharSequence status) {
+        // Include a meta tag so the WebView will not use a fixed viewport width of 980 px
+        return "<html><head><meta name=\"viewport\" content=\"width=device-width\"/>" +
+                HtmlConverter.cssStyleTheme() +
+                HtmlConverter.cssStylePre() +
+                "</head><body>" +
+                status +
+                "</body></html>";
+    }
+
+    private static String cssStyleTheme() {
+        if (K9.getK9MessageViewTheme() == K9.Theme.DARK)  {
```

Comment:
If you want to use the exit-early "pattern", invert the `if` condition. Otherwise please use an `else` statement.